New Bank/NewBank/newbank/server/Account.java

```diff
@@ -1,17 +1,29 @@
 package newbank.server;
 
 public class Account {
-	
+
 	private String accountName;
 	private double openingBalance;
 
 	public Account(String accountName, double openingBalance) {
 		this.accountName = accountName;
 		this.openingBalance = openingBalance;
 	}
-	
+
 	public String toString() {
 		return (accountName + ": " + openingBalance);
```

Comment:
this should return (accountName + ": " + openingBalance + " ") to have spaces between accounts when SHOWMYACCOUNT command is called

Reply:
Good spot, I've pushed the new change to this branch.